src/logic/operate.js

```diff
@@ -0,0 +1,17 @@
+const Big = require('big.js');
```

Comment:
Let's follow the required change commented in the package.json file and let's use the ES6 import syntax for this file since the proper library should not raise a linter error.